integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -223,3 +223,12 @@
 0.6.32
 
 - Add toggles to disable master/worker resource deployment
+
+0.6.33
```

Comment:
What should we do with this 0.6.33 changes that don't seem exist in branch-2.7? Do we really care?

Reply:
err i dont even know what this is about. are these bundled as part of our helm charts? if so, better to be accurate